utils/signatures/signatures.go

```diff
@@ -23,6 +23,13 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// DefaultPublicKeys are the public keys used to download content from Percona Platform.
+var DefaultPublicKeys = []string{
```

Comment:
We can keep this var private and encapsulate all logic inside `VerifySignatures`. So in other services we will just invoke this method `VerifySignatures(l *logrus.Entry, file string, signatures []string)` and it will use default keys or if `PERCONA_TEST_CHECKS_PUBLIC_KEY` is set then it will use the given value.

Reply:
done